### PR TITLE
fix: issue #326, delete empty volume

### PR DESCRIPTION
**Pull request by gabisavu-ionos**
## Checklist

- [ ] PR name added as appropriate (e.g. `feat:`/`fix:`/`doc:`/`test:`/`refactor:`)
- [ ] Tests added or updated
- [ ] Documentation updated
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [ ] Jira task updated

### CODE REVIEW

**Review by cristiGuranIonos — APPROVED** [review]

**Review by cristiGuranIonos — CHANGES_REQUESTED** [review]
Add to changelog.
Link github issue

**Review by cristiGuranIonos — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 6.3.3
+### Fixes
+- Fixes #326, now removing an s3_buckets block from an ionoscloud_k8s_cluster resource now trigger a change in terraform plan.
```
Comment (adeatcu-ionos) [inline CHANGELOG.md:3]: @gabisavu-ionos I think it should be:
`now removing a s3_buckets block from an ionoscloud_k8s_cluster resource triggers a change in the terraform plan.`

File: ionoscloud/resource_k8s_cluster_test.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ func TestAccK8sClusterBasic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckK8sClusterExists(K8sClusterResource+"."+K8sClusterTestResource, &k8sCluster),
 					resource.TestCheckResourceAttr(K8sClusterResource+"."+K8sClusterTestResource, "name", K8sClusterTestResource),
-					resource.TestCheckResourceAttr(K8sClusterResource+"."+K8sClusterTestResource, "k8s_version", "1.20.10"),
+					resource.TestCheckResourceAttr(K8sClusterResource+"."+K8sClusterTestResource, "k8s_version", "1.24.6"),
 					//resource.TestCheckResourceAttr(K8sClusterResource+"."+K8sClusterTestResource, "public", "true"),
```
Comment (adeatcu-ionos) [inline ionoscloud/resource_k8s_cluster_test.go:33]: @gabisavu-ionos what's up with all this commented code? I noticed that, besides this line, we have a lot of lines that are commented in `resource_k8s_cluster.go` file.

**Review by adeatcu-ionos — CHANGES_REQUESTED** [review]

File: ionoscloud/resource_k8s_cluster_test.go (hunk 2 above)
Comment (cristiGuranIonos) [reply ionoscloud/resource_k8s_cluster_test.go:33]: needs to be kept as a comment, will be added back in when private clusters are added back

**Pull request merged**